config/settings.yml

```diff
@@ -0,0 +1,6 @@
+links:
+  rails_tutorial:
+    home_page: "http://www.railstutorial.org/book"
+    help_page: "http://www.railstutorial.org/help"
+    book_page: "http://www.railstutorial.org/book"
+    
```

Comment:
![image](https://user-images.githubusercontent.com/65896888/91373158-dad1fd00-e83f-11ea-941d-8900177d1844.png)

Reply:
i already put an end line at line 6

Reply:
Dont write anything at line 6, even if space

Reply:
oh i see

Reply:
can you check again if I do it correctly ?